proposals/hackygolucky-EXPECTATIONS/EXPECTATIONS.md

```diff
@@ -0,0 +1,108 @@
+- Project representation on the Board 
+    - Community Representation should sit on the Board directly. Projects could share this representation. 
+    - Community representation on the Board: 3 seats guaranteed for community representation 
+- Board must establish Cross Project Council focused on the specific needs of the member projects 
+    - Allow projects to self-organize amongst themselves. Processes are defined and driven by project e.g. CPC is not structured by the Board 
+- Non-corporate member representation in budget/finance subcommittee 
+- Projects values should drive foundation focus and priority 
+- No "pay to play" in technical participation 
+- Earmarking allowed with flexibility in budgeting for concerns and needs of individual projects 
+- Value satisfaction: regular feedback from projects 
+- Explicit acknowledgement and documention for the ability of projects to leave the foundation 
+- Foundation should document a clear process for trademark usage to enable community not-for-profit efforts
+- Communicated scope for the foundation and board
+  - Annual reviews of staff, NPS, board, programs, support services to projects for accountability and incremental improvement
+  - The Foundation staff, executive director, and board should follow a "servant leadership" style.
+  - Non-corporate member representation in budget / finance subcommittee
+  - Projects values should drive foundation focus and priority
+  - No "pay to play" in technical participation
+  - Earmarking allowed with flexibility in budgeting for concerns and needs of individual projects
+  - Value satisfaction: regular feedback from projects
+- Executive Director required to work directly with projects to gather strategy and budget needs for upcoming roadmaps including but not limited to: 
+  - Brand management per Project
+  - Budget Allocation per Project
+  - Resource Allocation per Project
+
+### Communicated expectations
+
+##### Expectations of the Foundation
+- Protect Intellectual Property (IP) including trademarks 
+    - Enable the project communities to use the trademarks 
+    - Educate members, new and old, about proper use of the projects' IP 
+
+- Provide Legal Advice and Legal Best Practices 
+    - Provide legal resources and representation that support the needs of the project 
+    - Create a well-defined communications path for project legal needs 
+
+- Software / Hardware resource facilitation (for use by the project) 
+    - Examples: 
+        - OKTA Identity Service for Individual Membership 
+        - Headless CMS for website 
+        - Translation and Internationalization Platform 
+        - Have asked for approval to buy mac minis….  
+
+- Provide Ongoing Supporting Personnel to help manage community and technical efforts 
+    - Examples: 
+        - Community Manager 
+        - Education Manager 
+
```

Comment:
Not sure where this would actually go, but @boneskull pointed out that another thing the projects need help with is identifying/capturing which companies are using their tools internally. this benefits the projects and foundation from a marketing standpoint, and helps generate a list of potential new foundation members.

Reply:
I think after [the recent event-stream npm incident](https://blog.npmjs.org/post/180565383195/details-about-the-event-stream-incident), what @mhdawson said is doubly true. Having: "Our package is used internally by $BIG_COMPANY" may make that project a target for this kind of attack. 😕 I really wish this wasn't the case.